### PR TITLE
ISPN-13979 Simple JMX remoting

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-13979

### CODE REVIEW

File: server/runtime/src/main/java/org/infinispan/server/security/ElytronJMXAuthenticator.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      NameCallback name = new NameCallback("username");
+      PasswordCallback password = new PasswordCallback("password", false);
+      try {
+         callbackHandler.handle(new Callback[]{name, password});
+         SecurityIdentity securityIdentity = securityDomain.authenticate(name.getName(), new PasswordGuessEvidence(password.getPassword()));
+         subject.getPrincipals().add(securityIdentity.getPrincipal());
+         securityIdentity.getRoles().forEach(r -> subject.getPrincipals().add(new RolePrincipal(r)));
+         return true;
+      } catch (IOException | UnsupportedCallbackException e) {
+         Server.log.error("JMX authentication error", e);
```
Comment (pruivo) [inline server/runtime/src/main/java/org/infinispan/server/security/ElytronJMXAuthenticator.java:54]: should be a method in `Log` class

File: server/runtime/src/main/server/bin/common.sh [hunk 2]
```diff
@@ -3,10 +3,17 @@
 # Use --debug to activate debug mode with an optional argument to specify the port.
 # Usage : server.sh --debug
 #         server.sh --debug 9797
-
 # By default debug mode is disabled.
 DEBUG_MODE="${DEBUG:-false}"
 DEBUG_PORT="${DEBUG_PORT:-8787}"
+
```
Comment (pruivo) [inline server/runtime/src/main/server/bin/common.sh:9]: shouldn't the `.bat` files be updated as well?

Comment (pruivo) [reply server/runtime/src/main/server/bin/common.sh:9]: fair enough :+1:

**Comment by tristantarrant** [comment]
Tests fixed

**Comment by pruivo** [comment]
merged! thanks @tristantarrant !

**Pull request merged**